internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -727,3 +727,25 @@ func (r *Reconciler) reconcileReferencedTemplate(ctx context.Context, in reconci
 
 	return nil
 }
+
+// createErrorWithoutObjectName removes the name of the object from the error message. As each new Create call involves and
+// object with a unique generated name each error appears to be a different error. As the errors are being surfaced in a condition
+// on the Cluster, the name is removed here to prevent each creation error from triggering a new reconciliation.
+func createErrorWithoutObjectName(err error, obj client.Object) error {
+	out := ""
+	var statusError *apierrors.StatusError
+	if errors.As(err, &statusError) {
+		if statusError.Status().Details != nil {
+			for _, cause := range statusError.Status().Details.Causes {
+				out = fmt.Sprintf("%s %s: %s: %s", out, cause.Type, cause.Field, cause.Message)
+			}
+			out = fmt.Sprintf("%s.%s: %s.", statusError.Status().Details.Kind, statusError.Status().Details.Group, out)
```

Comment:
```suggestion
			out = fmt.Sprintf("%s.%s: %s", statusError.Status().Details.Kind, statusError.Status().Details.Group, out)
```
nit, can we please drop the trailing dot (usually error messages don't have a trailing dot)